projects/packages/videopress/src/client/block-editor/blocks/video/components/player/style.native.scss

```diff
@@ -10,3 +10,21 @@
 	width: 100%;
 	z-index: 1;
 }
+
+.videopress-player__loading {
```

Comment:
There is no difference between dark and light modes for the embed processing screen. I opted to keep the loading view as close to the embed processing view so I didn't add light/dark versions.